scripts/release.sh

```diff
@@ -258,13 +258,6 @@ main() {
       log_warning "login failed, retrying"
     done
 
-    for TARGET_ARCH in "amd64" "arm64" "ppc64le" "s390x"; do
```

Comment:
Would be good to test if you can push multi-arch image without pushing each arch. Workaround would be to just push images but avoid tagging them.

Reply:
Ahh bad news, yeah `docker manifest` will only ever work with the registry images and can't do any magic that would allow us not to push the multi arch only.

Unfortunately even `docker buildx` works pretty much the same way, just with simpler ux.

So yeah we could push and not tag but actually reviewing kubernetes docs they are still publishing individual arch images:

> https://kubernetes.io/releases/download/
> All container images are available for multiple architectures, whereas the container runtime should choose the correct one based on the underlying platform. It is also possible to pull a dedicated architecture by suffixing the container image name, for example registry.k8s.io/kube-apiserver-arm64:v1.26.0. All those derivations are signed in the same way as the multi-architecture manifest lists.

So actually I don't think we can simplify the image pushing at all?  Your call if we just close this or continue with just the refactoring I mentioned above.